packages/js/src/preferences/helpers.ts

```diff
@@ -19,13 +19,19 @@ export const updatePreference = async ({
     emitter.emit('preference.update.pending', {
       args,
       data: args.preference
-        ? new Preference({
-            ...args.preference,
-            channels: {
-              ...args.preference.channels,
-              ...channelPreferences,
+        ? new Preference(
+            {
+              ...args.preference,
+              channels: {
+                ...args.preference.channels,
+                ...channelPreferences,
+              },
             },
-          })
+            {
```

Comment:
Ditto (see Notification) class. Also here they are passed as an object whereas in the other classes they are discrete arguments in the constructor.